docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -82,19 +82,34 @@ In the webhook settings page, make sure that the webhook will only send Issue Up
 
 We recommend installing Teleport plugins alongside the Teleport Proxy. This is an ideal
 location as plugins have a low memory footprint, and will require both public internet access
-and Teleport Auth access. We currently only provide linux-amd64 binaries, you can also
-compile these plugins from [source](https://github.com/gravitational/teleport-plugins/tree/master/access/jira).
+and Teleport Auth access. 
 
-```code
-$ wget https://get.gravitational.com/teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-$ tar -xzf teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-$ cd teleport-access-jira/
-$ ./install
-$ which teleport-jira
-# /usr/local/bin/teleport-jira
-```
-
-Run `./install` in from 'teleport-jira' or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
+<Tabs>
+<TabItem label="Download">
+  ```code
+  $ curl -L https://get.gravitational.com/teleport-access-jira-v(=teleport.version=)-linux-amd64-bin.tar.gz
+  $ tar -xzf teleport-access-jira-v(=teleport.version=)-linux-amd64-bin.tar.gz
+  $ cd teleport-access-jira
+  $ ./install
+  ```
+</TabItem>
+<TabItem label="From Source">
+  To install from source you need `git` and [`go`](https://go.dev/) installed.
+
+  ```code
+  # Checkout teleport-plugins
+  $ git clone https://github.com/gravitational/teleport-plugins.git
+  $ cd teleport-plugins/access/jira
+  $ make
+  ```
+Run `./install` from `teleport-jira` or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
+</TabItem>
+<TabItem label="Docker">
+  ```code
+  $ docker pull quay.io/gravitational/access-plugin-jira:(=teleport.version=)
```

Comment:
We should make sure that no value of `(=teleport.version=)` will point to a nonexistent container image tag. It might be safer to add a variable key to `config.json` for the latest `access-plugin-jira` plugin version.

Reply:
This also applies to the other `docker pull` commands added in this PR.

Reply:
Good catch. I've updated the `config.json` to reference the latest version of teleport that also happens to be the first plugin version that has a built image. 

Doesn't seem like this is actively being set in releases but atleast it'll now not reference a missing image.